docs/CONTRIBUTING.md

```diff
@@ -49,6 +49,19 @@ Please follow [Venice Workspace Setup](./dev_guide/workspace_setup.md) and
    submitting a pull request. Large features which have never been
    discussed are unlikely to be accepted.
 
+## Contributor Sync Meeting 
+
+Venice does Contributor sync meeting for the contributors to come togather in a virtual meeting.
+
+1. Meeting Frequency : Bi-weekly on Tuesday at 8 AM Pacific, 11 AM Eastern, 5 PM Central Europe
+2. Meeting Co-ordination : Please join the #contributor-sync channel in the 
+     [public Venice Slack](https://communityinviter.com/apps/venicedb/venice) 
```

Comment:
Let's use the alias: http://slack.venicedb.org in case community inviter stops working well in the future, there'll be fewer links to tweak later on.

Reply:
Good Idea. Done.